src/System.Runtime/tests/System/ArrayTests.cs

```diff
@@ -1931,6 +1931,8 @@ public static void GetEnumerator_Invalid(Array array)
 
         public static IEnumerable<object[]> IndexOf_SZArray_TestData()
         {
+            // TODO: Uncomment when https://github.com/xunit/xunit/issues/1771 is fixed
```

Comment:
It feels like this should go near the actual commented out code.

Reply:
Also, a TODO is not a good way to remember something -- maybe you could have a tracking issue for these various odds and ends.

Reply:
at minimum put `[ActiveIssue(` in the comment so it is easier to grep

Reply:
I think a TODO is perfectly fine. While updating the runner I myself grepped for "xunit/xunit" and a few other keywords (i.e. T dummy) which listed all the places where changes were necessary. That said, I don't mind opening an issue here in corefx (as there's already one on the xunit side) if you think that's a better approach.

Reply:
https://github.com/dotnet/corefx/issues/31874

Reply:
We periodically ask our vendors to make a pass looking for tests that should not be disabled anymore. It's more accurate if they only have one or two patterns to look for.

Reply:
ok good to know. done.